src/syscheckd/src/db/tests/db/ComponentTest/fileInterface/fileTest.cpp

```diff
@@ -246,27 +266,33 @@ TEST_F(DBTestFixture, TestFimDBFileInodeSearch)
 
     EXPECT_NO_THROW(
     {
-        fim_db_file_update(fileFIMTest1->toFimEntry(), callback_data_added);
-        fim_db_file_update(fileFIMTest2->toFimEntry(), callback_data_added);
-        fim_db_file_update(fileFIMTest3->toFimEntry(), callback_data_added);
+        auto result = fim_db_file_update(fileFIMTest1->toFimEntry(), callback_data_added);
+        ASSERT_EQ(result, FIMDB_OK);
+        result = fim_db_file_update(fileFIMTest2->toFimEntry(), callback_data_added);
+        ASSERT_EQ(result, FIMDB_OK);
+        result = fim_db_file_update(fileFIMTest3->toFimEntry(), callback_data_added);
+        ASSERT_EQ(result, FIMDB_OK);
         char *test;
         test = strdup("/etc/wgetrc");
         callback_context_t callback_data;
         callback_data.callback = callbackTestSearchPath;
         callback_data.context = test;
         try
```

Comment:
I think, that this and the successive try catch is totally unnecessary and doesn't make any sense.

Reply:
Done !